cfme/common/provider.py

```diff
@@ -101,6 +103,46 @@ def deployment_helper(self, deploy_args):
         """ Used in utils.virtual_machines and usually overidden"""
         return {}
 
+    @property
+    def default_endpoint(self):
+        try:
+            return self.endpoints['default']
+        except (KeyError, AttributeError):
```

Comment:
oh, I just forgot to remove it from CloudProvider when moved it to BaseProvider.